Code/src/main/java/nl/utwente/viskell/ui/components/InputAnchor.java

```diff
@@ -103,6 +103,10 @@ public boolean hasConnection() {
     public Type getType() {
         return this.type;
     }
+    
+    public void setType(Type t) {
```

Comment:
This setter is a bit scary - it could use some Javadoc that described who is allowed to call it, and when.